L/libgraphicsmagic/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libgraphicsmagic"
+version = v"1.3.36"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("ftp://ftp.icm.edu.pl/pub/unix/graphics/GraphicsMagick/1.3/GraphicsMagick-1.3.36.tar.gz", "1e6723c48c4abbb31197fadf8396b2d579d97e197123edc70a4f057f0533d563")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd GraphicsMagick-1.3.36/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --enable-shared
+make
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl")
+]
```

Comment:
Is this supposed to work only on Linux?  If so, you can simplify with
```suggestion
platforms = filter!(Sys.islinux, supported_platforms(; experimental=true))
```
otherwise we can try to build for all supported platforms with
```suggestion
platforms = supported_platforms(; experimental=true)
```

Reply:
I think macOS is close enough, Windows looks very different (at least on their installation page)